completions/bash/crio

```diff
@@ -110,6 +110,7 @@ h
 --selinux
 --separate-pull-cgroup
 --signature-policy
+--signature-policy-namespace-root
```

Comment:
I am not a huge fan of the wording here but I don't have anything else to suggest. If I or no one else can think of anything better then I am fine with this, but I find the name clunky as-is

Reply:
What about `--signature-policy-dir` and the description can mention that it is where namespace separated signature policies are stored in the format `namespace.json`. So the final policy path used will be `signaturePolicyDir/Namespace.json`

Reply:
yeah I think that's a clean approach